notebooks/1_table_oriented.ipynb

```diff
@@ -0,0 +1,484 @@
+{
```

Comment:
<p>The Note seems a little strange to me. I get what you are saying but for a new user it doesn't really explain much, so probably best to remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>On overcoming too much links towards _future_ tutorials and due to the potential confusion for new users, I removed this note.</p><p> </p>